crowdin_api/api_resources/abstract/resources.py

```diff
@@ -5,12 +5,16 @@
 
 
 class BaseResource(metaclass=ABCMeta):
-    def __init__(self, requester: APIRequester, page_size=25):
+    def __init__(self, requester: APIRequester, project_id: int, page_size=25):
```

Comment:
This configuration should be on the highest configuration level like the `TOKEN`, `ORGANIZATION` options.

Reply:
Also, it should be documented in the Readme

Reply:
> should be on the highest configuration level

sorry, overlooked it, it's already there